src/containers/WelcomePage/WelcomePageSearch.jsx

```diff
@@ -59,7 +58,7 @@ const WelcomePageSearch = ({ history, locale }) => {
 
   if (error) {
     handleError(error);
-    return `Error: ${error.message}`;
+    return <DefaultErrorMessage />;
```

Comment:
Vi kan ikkje gjøre dette akkurat her, fordi sjølv om dette søket feiler så funker resten av sida. Då er det bedre å vise en generisk feilmelding og la brukeren klikke seg inn på et fag,

Reply:
Viser den samme nå uten linker og gif. Prøvde å legge inn den samme i MastheadSearch, men den passet ikke inn. Det går kanskje an å lage feilhåndtering for søkefelt i frontend-packages, men vet ikke hvordan det bør se ut.